### ISSUE TITLE
Offline reading is not a property of a document

### ISSUE THREAD

**Issue by jar398** [w3c/dpubwg-charter#25]
The scope section for the most part says that the project is merely to define a new document type (in the sense of MIME type), although I suppose it does not commit to that. This is a much more straightforward and conventional idea than the rhetoric found in the goals section seems to say, so I think both sections should be rewritten to make this clear. However, one requirement makes no sense in this interpretation:

"A Web Publication must be available and functional while the user is offline. A user should, as much as possible, have a seamless experience of interacting with a Web Publication regardless of their network connection. We make no distinction between online and offline when defining Web Publications."

There is absolutely nothing a publication, document, or "Web Publication" can do to make itself available and functional. The "must" has to apply to agents such as readers and distribution networks. The wording is a kind of passive voice, or agentless construction, that is likely to be confusing both to readers of the charter and to WG members down the road. I recommend rewriting this requirement (and perhaps the entire section) to make it clear what sorts of things are going to be constrained by the proposed specification: documents of the new type, readers, generators, distribution networks, etc.

In other words: any specification specifies that a something X should (must) have property P. (And if the specification is to be actionable, P has to be testable somehow.) Be very clear on what X is.

**Comment by WSchindler** [comment]
Let me try to find to integrate the argument of @jar398:
"_Any software agent consuming and rendering_ a Web Publication must _ensure that it is_ available and functional while the user is offline." (my additions are in italics)

**Comment by lrosenthol** [comment]
@jar398 and @WSchindler 
While I agree that many aspects of "going offline" can/will be handled by the UA, there **are** things that a WP can do as well.  Business rules/logic that determine whether about what to present when offline (especially in the context of a live data feed), for example.   There is nothing that a UA could do in that case.

**Comment by WSchindler** [comment]
When your content consists in part of live data drawn from the web, I absolutely agree with you that a WP spec should define how to handle this aspect in an offline scenario. This would be thus be part of defining an aspect of a WP.

**Comment by bfoleary** [comment]
I've been charged with editing the draft, mostly for grammar and style, sometimes for content. For the fourth bullet, I'm proposing the following combined language:

- When defining Web Publications, we make no distinction between online and offline reading. Regardless of their network connections, users should have a seamless experience interacting with a Web Publication. Any software agent consuming and rendering a Web Publication must ensure that it is available and functional while the user is offline.

**Comment by lrosenthol** [comment]
I can live with that.

**Comment by jar398** [comment]
Fine by me
